### DISCUSSION TITLE
How to parameter the profil

### DISCUSSION THREAD

**Discussion by faon-92** [SubtitleEdit/subtitleedit#6978] [Q&A]
Hi everyone,

Concerning all the parameters of a profil, I'm wondering:

1. if my projet don't need to set a parameter, e.g. max. words/min, I just put a very high number, or I can input zero, which mean to ignore this parameter? 
2. what is the impact of optimal chars/sec, compared to max. chars/sec which will be colors if I exceed?

thanks for your help
<img width="291" alt="SubtitleEdit_msmD0J5W7n" src="https://github.com/SubtitleEdit/subtitleedit/assets/127494540/bd20e5b9-83ba-407d-8b95-1afecc359cef">

**Comment by niksedk** [comment]
1) If you don't want to use max. words/min just put a high number. The default value is already a bit high.

2) Optimal chars/sec is used when re-calculating a new duration for a subtitle - e.g. Tools - Adjust duration - re-calculate
Max chars/sec is used for validation and min. duration of lines.

**Comment by faon-92** [comment]
Many thanks.
I have other questions about others subjects, would you like me to back to
the githun site and open a new discussion?

On Fri, Jun 2, 2023 at 12:01 PM Nikolaj Olsson ***@***.***>
wrote:

>
>    1.
>
>    If you don't want to use max. words/min just put a high number.
>    2.
>
>    Optimal chars/sec is used when re-calculating a new duration for a
>    subtitle - e.g. Tools - Adjust duration - re-calculate
>    Max chars/sec is used for validation and min. duration of lines.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SubtitleEdit/subtitleedit/discussions/6978#discussioncomment-6070100>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6MWTDBWJ642NHZZFCJ7E6DXJG2WJANCNFSM6AAAAAAYYBPJRA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Feng Sridi

**Reply by niksedk** [reply]
Yes, this is probably best

**Comment by faon-92** [comment]
Can I ask you a newbie question?

how I should do to apply dialogue style and continuation style to my
subtitles?
[image: SubtitleEdit_NGLCviW8AT.png]

On Fri, Jun 2, 2023 at 12:17 PM Nikolaj Olsson ***@***.***>
wrote:

> Yes, this is probably best
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SubtitleEdit/subtitleedit/discussions/6978#discussioncomment-6070234>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6MWTDBZRKORQGGYBDQGQBTXJG4VFANCNFSM6AAAAAAYYBPJRA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Feng Sridi
